locache.js

```diff
@@ -104,6 +108,7 @@
     // Internal utility functions
     // --------------------
 
+    // @Deprecated
```

Comment:
I'd rather see this just deleted if it isn't used internally anymore. It isn't part of the public API, so we don't need to support it.